enhancements/console/quick-starts.md

```diff
@@ -151,6 +155,30 @@ resource actions. In order for the user to see the Quick Start and
 take its tour, his access review needs to pass all the listed
 resources actions.
 
+#### Quick Start Internationalization
+
+OpenShift can be toggled between multiple languages. The
+Quick Start CRD `label` contains a `language` field to specify what
+language the Quick Start is in. CRDs without this label will be treated
+as English CRDs. OpenShift can display the correct
```

Comment:
I'd say specifically "Quick Starts" here so it's clear this is specific to Quick Starts and not other CRDs.

```suggestion
language the Quick Start is in. Quick Starts without this label will be treated
as English Quick Starts. OpenShift can display the correct
```